src/vos/tests/vts_pool.c

```diff
@@ -527,7 +527,7 @@ pool_open_excl_test(void **state)
 
 	print_message("open shall fail upon existing EXCL create opener\n");
 	ret = vos_pool_create(arg->fname[0], uuid, VPOOL_256M, 0, VOS_POF_EXCL,
-			      &arg->poh[0]);
+			      0, &arg->poh[0]);
```

Comment:
This is the defect, the 0 'meta_sz' should be put before the 'VOS_POF_EXCL' flag.